packages/components/inputs/time-input/src/time-input-body.styles.js

```diff
@@ -153,100 +102,86 @@ const getInputContainerStyles = (props) => {
     ...props.theme,
   };
 
-  return css`
+  const baseStyles = css`
+    appearance: none;
+    background-color: ${overwrittenVars.backgroundColorForInput};
+    border: 1px solid ${overwrittenVars.borderColorForInput};
+    border-radius: ${overwrittenVars.borderRadiusForInput};
+    box-sizing: border-box;
+    color: ${overwrittenVars.fontColorForInput};
     width: 100%;
+    height: ${vars.sizeHeightInput};
     align-items: center;
     display: flex;
-    font-size: ${overwrittenVars[designTokens.fontSizeForInput]};
+    font-size: ${overwrittenVars.fontSizeDefault};
     font-family: inherit;
+    transition: border-color ${overwrittenVars.transitionStandard},
+      box-shadow ${overwrittenVars.transitionStandard};
+
+    &:focus-within {
+      border-color: ${overwrittenVars.borderColorForInputWhenFocused};
+      box-shadow: inset 0 0 0 2px
+        ${overwrittenVars.borderColorForInputWhenFocused};
+    }
+
+    &:focus,
+    &:hover {
+      border-color: ${overwrittenVars.borderColorForInputWhenFocused};
+    }
   `;
-};
 
-const mapProps = (props) => {
```

Comment:
Removed because these were added to the shared styles (referenced in comment above).